automatic/displayfusion/update.ps1

```diff
@@ -19,8 +19,8 @@ function global:au_SearchReplace {
 
 function global:au_GetLatest {
   $urls = @(
-    "https://www.binaryfortress.com/Data/Download/?package=displayfusion&log=101&beta=0"
-    "https://www.binaryfortress.com/Data/Download/?package=displayfusion&log=101&beta=1"
+    "https://www.binaryfortress.com/Data/Download/?package=displayfusion&log=101&beta=0" # This URL redirects to the stable version
+    #"https://www.binaryfortress.com/Data/Download/?package=displayfusion&log=101&beta=1" # This URL now returns a 404 causing the update to fail
```

Comment:
I don't think just commenting this would line would be the best way to handle this.

If I remember correctly, this link will be valid in the future again when there is a beta release available.
Would you be in a position to instead update the aquiring of the url to just ignore that stream error instead?

Reply:
Work on handling the commented out url better is postponed to some time in the future.